scss/_patterns_modal.scss

```diff
@@ -35,6 +35,7 @@
     right: $sph-inner--large;
     top: $spv-inner--large;
     width: auto;
+    padding-top: 0;
```

Comment:
I think the CI is failing because this is not in alphabetical order. Do you have stylelint set up in VS code?

Reply:
I thought so, but I guess not. Will look into it now.